SStream.c

```diff
@@ -66,8 +66,7 @@ void printInt64Bang(SStream *O, int64_t val)
 				SStream_concat(O, "#-0x%"PRIx64, (uint64_t)val);
 			else
 				SStream_concat(O, "#-0x%"PRIx64, (uint64_t)-val);
-		}
-		else
+		} else
```

Comment:
changing coding style? is the capstone coding style documented anywhere?

Reply:
no, we always use Linux kernel style. i just updated HACK.TXT with this info: https://github.com/aquynh/capstone/commit/864595c3559a27fbcddc32b15f59db9e74342d2e